independent-publisher-connectors/TSheets Quickbooks Time/apiDefinition.swagger.json

```diff
@@ -0,0 +1,654 @@
+                        "default": "all",
+                        "enum": [
+                            "regular",
+                            "pto",
+                            "paid_break",
+                            "unpaid_break",
+                            "all"
+                        ],
+                        "x-ms-visibility": "advanced",
+                        "description": "Only timesheets linked to a jobcode of the given type are returned."
+                    },
+                    {
+                        "name": "modified_before",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "x-ms-summary": "Modified Before",
+                        "description": "in ISO 8601 format (YYYY-MM-DDThh:mm:ss\u00b1hh:mm)"
+                    },
+                    {
+                        "name": "modified_since",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "x-ms-summary": "Modified Since",
+                        "description": "Only timesheets modified since this date/time will be returned, in ISO 8601 format (YYYY-MM-DDThh:mm:ss\u00b1hh:mm)."
+                    },
+                    {
+                        "name": "supplemental_data",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "x-ms-summary": "Supplemental Data",
+                        "default": "yes",
+                        "enum": [
+                            "yes",
+                            "no"
+                        ],
+                        "x-ms-visibility": "advanced",
+                        "description": "yes' or 'no'. Default is 'yes'. Indicates whether supplemental data should be returned."
+                    },
+                    {
+                        "name": "per_page",
+                        "in": "query",
+                        "required": false,
+                        "type": "integer",
+                        "x-ms-summary": "Per Page",
+                        "description": "Represents how many results you'd like to retrieve per request.",
+                        "x-ms-visibility": "advanced"
+                    },
+                    {
+                        "name": "page",
+                        "in": "query",
+                        "required": false,
+                        "type": "integer",
+                        "x-ms-summary": "Page",
+                        "description": "Represents the page of results you'd like to retrieve.",
+                        "x-ms-visibility": "advanced"
+                    }
+                ]
+            }
+        },
+        "/notifications": {
+            "get": {
+                "responses": {
+                    "default": {
```

Comment:
@artesianst default responses is not allowed. Please define a proper response, please see [here](https://learn.microsoft.com/en-gb/connectors/custom-connectors/coding-standards#responses)